<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Mathias Soeken &lt;msoeken@informatik.uni-bremen.de&gt;</author>
 <class>LatexCommandWidget</class>
 <widget class="QWidget" name="LatexCommandWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>286</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Define LaTeX Environments and Commands for Kile</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageEnvironments">
      <attribute name="title">
       <string>Environments</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="environments">
         <property name="whatsThis">
          <string>List of known environments with a lot of additional information that Kile can potentially use. You can add your own environments, which will then be recognized by autocompletion of environments; 'Smart Newline' and 'Smart Tabulator', for example. Note that you can only edit and delete user-defined environments.</string>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Environment</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Starred</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>EOL</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Math</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tab</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Option</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Parameter</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageCommands">
      <attribute name="title">
       <string>Commands</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="commands">
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Command</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Starred</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Option</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Parameter</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="whatsThis">
         <string>Add a new environment.</string>
        </property>
        <property name="text">
         <string>&amp;Add...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="whatsThis">
         <string>Delete a user-defined environment.</string>
        </property>
        <property name="text">
         <string>De&amp;lete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editButton">
        <property name="whatsThis">
         <string>Edit a user-defined environment.</string>
        </property>
        <property name="text">
         <string>&amp;Edit...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showOnlyUserDefined">
     <property name="text">
      <string>Show only user-defined environments and commands</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
